src/javascript/app/common/deriv_banner.js

```diff
@@ -0,0 +1,50 @@
+const DerivBanner = (() => {
+    let $deriv_banner_countainer,
+        $multiplier_banner_container,
+        $deriv_banner_close_buttons;
+
+    const onLoad = () => {
+        const is_deriv_banner_dismissed = localStorage.getItem('is_deriv_banner_dismissed');
+        $deriv_banner_countainer = document.getElementById('deriv_banner_container');
+        $multiplier_banner_container = document.getElementById('multiplier_banner_container');
```

Comment:
can we move this assignments inside the if condition since they're not needed otherwise? also let's use our common function getElementByID to handle in case the element is not found

Reply:
Yes sure